src/main/kotlin/de/novatec/betting/game/teams/tf/TeamsTf.kt

```diff
@@ -0,0 +1,19 @@
+package de.novatec.betting.game.teams.tf
+
+import de.novatec.betting.game.openliga.model.OLTeam
+import de.novatec.betting.game.teams.model.Team
+import de.novatec.betting.game.teams.model.Teams
+import javax.inject.Singleton
+
+/** Transformer class that maps [OLTeam] to the internal [Team] model. */
+@Singleton
+class TeamsTf {
+
+    /** Maps a [List] of [OLTeam]s to a [Teams] instance.
+     *
+     * @param olTeams [List] of [OLTeam]s
+     * @return a [Teams] instance containing [Team]s
+     */
+    fun olTeamsToTeams(olTeams: List<OLTeam>): Teams =
```

Comment:
Da die Argumente fast alles STrings sind, würde ich dazu tendieren, hier named arguments zu verwenden. Dan gibt's auch kein Problem, wenn in der Data Class mal was gedreht wird.
```
fun olTeamsToTeams(olTeams: List<OLTeam>) =
        olTeams.map {
            Team(
                id = it.teamId,
                name = it.teamName,
                shortName = it.shortName,
                logo = it.teamIconUrl
            )
        }.toList()
```